moveit_core/utils/CMakeLists.txt

```diff
@@ -2,34 +2,37 @@ set(MOVEIT_LIB_NAME moveit_utils)
 
 add_library(${MOVEIT_LIB_NAME}
   src/lexical_casts.cpp
-  src/xmlrpc_casts.cpp
+  # src/xmlrpc_casts.cpp # TODO: solve the conflicts with xmlrpc dependencies
```

Comment:
I think `xmlrpc_casts.cpp` will not be used anymore since XML-RPC is not supported in ROS 2.
`parseDouble()` and `isArray()` should be redundant in the future but we have to check where`isStruct()` is used and if we need to reimplement it somehow.
Let's remove `xmlrpc_casts.cpp` and add TODOs everywhere where it is used.
These parts will need to be changed anyway since the same missing XML-RPC dependencies are used.